components/admin_console/jobs/table.jsx

```diff
@@ -14,7 +14,7 @@ class JobTable extends React.PureComponent {
         /**
          * Used for formatting dates
          */
-        intl: intlShape.isRequired,
+        intl: PropTypes.any,
```

Comment:
Hmm, since the `shallowWithInject()` have littel tricky cloning,

```tsx
shallowWithIntl(<MyIntlComponent/>
↓↓↓
shallow(<MyIntlComponent intl={defaultIntl} ... />
```

So it should able to rendered without intl object once.

Marking isRequired is more helpful for catching runtime errors, but it causes the test to fail.

Reply:
Oh yeah, I think I've hit that issue before with some other props. It makes sense to not have it required then.